deploy/overlays/stanford/memdb.json

```diff
@@ -0,0 +1,38 @@
+{
+  "importName": "Private_DC_Template",
+  "provenanceUrl": "https://www.datacommons.org/",
+  "dataDownloadUrl": "https://www.datacommons.org/",
+  "rootSvg": "g/Template",
+  "statVarGroups": {
+    "g/Template": {
+      "childStatVarGroups": [
+        {
+          "id": "g/Template_SVG_Level_1",
+          "specializedEntity": "Template SVG Level 1"
+        },
+      ],
+      "childStatVars": [
```

Comment:
As a minimal template, maybe just retain "Template_SV_Level_1" here?

Also should change the tmcf file in gcs to have ".tmcf" extension

Reply:
Done and done. Please take a look :)